<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tarefas de Hoje</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="dgvDadosHoje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>titulo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nota</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>data</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>status</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
